data/mods/Magiclysm/Spells/attunements/Alchemist.json

```
@@ -1,4 +1,29 @@
 [
+  {
+    "id": "alchemist_make_potion_starter",
+    "type": "SPELL",
+    "name": { "str": "Infuse water" },
+    "description": "Infuse water with your own mana, and use ethanol from the air to create the alchemic base.  Sometimes reaction is too strong, and the result can pour on the ground.",
```

some parts are a bit unclear
```suggestion
    "description": "Infuse water with ethanol from the air and your own mana to create some potion starter.  Sometimes it might overreact and overflow.",
```

---

Chill, its not related to this PR
If you want to fix such stuff its better to onen a new pull request with them

---

Oh, its another pr
My bad 😅